website/docs/how-to/how-to-create-service-accounts.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: How to create service accounts
+---
+
+:::info availability
+
+Service accounts is an upcoming enterprise feature, and is scheduled to become available in one of the next few releases.
+
+:::
+
+[Service accounts](../reference/service-accounts.md) enable Unleash admins to create accounts that act as users and respect the same set of permissions, however they do not have a password and cannot log in to the Unleash UI. Instead, they are intended to be used to access the Unleash API programatically.
+
+## Step 1: Navigate to the service accounts page {#step-1}
+
+Navigate to the _Service accounts_ page in the admin UI (available at the URL `/admin/service-accounts`). Use the "Settings Wheel" on the navigation menu and select "Service accounts".
+
+![The admin UI navigation "Settings Wheel" submenu with the Service accounts item highlighted.](/img/service-account-1.png)
+
+## Step 2: Click the "new service account" button {#step-2}
+
+On the top right corner, use the "new service account" button to open the "new service account" form.
```

Comment:
In general, I'd suggest leaving the specific location of elements out of guides. Primarily so that we don't need to update it if we change the layout.

```suggestion
Use the "new service account" button to open the "new service account" form.
```